dev/com.ibm.ws.logging/resources/com/ibm/ws/logging/internal/resources/LoggingMessages.nlsprops

```diff
@@ -156,6 +156,14 @@ JSON_FIELDS_INVALID_CHARACHTER=TRAS3011W: An invalid character was provided in t
 JSON_FIELDS_INVALID_CHARACHTER.explanation= Only letters, numbers, and : are accepted in the JSON logging field name mapping property
 JSON_FIELDS_INVALID_CHARACHTER.useraction= Change your JSON logging field name mapping property value to remove invalid characters
 
+LOG_ROLLOVER_START_TIME_FORMAT_WARNING=TRAS3012W: The log rollover start time property is incorrectly formatted. The value will be set to a default of '00:00', representing midnight.
+LOG_ROLLOVER_START_TIME_FORMAT_WARNING.explanation=The rollover start time property must be formatted HH:MM, with 00:00 representing midnight.
+LOG_ROLLOVER_START_TIME_FORMAT_WARNING.useraction=The rollover start time property must be formatted HH:MM, with 00:00 representing midnight.
```

Comment:
should the message explanation specify 24 vs 12-hour notation? I'm assuming it's 24-hour not 12, if 00:00 is midnight. Is that correct?

Reply:
Yes, this should be 24 hour, which we document in the [metatype.properties](https://github.com/OpenLiberty/open-liberty/pull/18429/files/0e8c0d5fb4b8683a70163340fdc91f3568d19aa7#diff-d259bd47effd0bed61713d1a6c4e90461d5700056071aaa43b3dd8c0b5430c88).